Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -190,7 +190,7 @@ protected void testGetActiveParticipantSource(LogEventRequestType request, Boole
         }
 
         assertEquals("NetworkAccessPointID mismatch", ipOrHost, sourceActiveParticipant.getNetworkAccessPointID());
-        assertEquals("SourceActiveParticipant requestor flag mismatch", isRequesting,
+        assertEquals("SourceActiveParticipant requestor flag mismatch", Boolean.TRUE,
```

Comment:
yes. "UserIsRequestor" should be "true" even if isRequesting holds a value of "false" or "true"